source/templates/committer-welcome.txt

```diff
@@ -0,0 +1,67 @@
+To: private@[PROJECT].a.o, NewCommitterID@apache.org
+Subject: Welcome, New Committer!
+
+[Committer Name] as you know, the ASF Infrastructure has set up your
+committer account with the username `[apacheID]`
+
+[If your project uses SVN]
```

Comment:
I think projects using SVN are (vast) minority now - maybe even almost none of our projects uses SVN. It would be worthwile to reverse those two sections to make sure that GitHub is the first on the list.